en/orm/retrieving-data-and-resultsets.rst

```diff
@@ -164,7 +164,8 @@ methods will let you re-use your queries and make testing easier.
 
 By default queries and result sets will return :doc:`/orm/entities` objects. You
 can retrieve basic arrays by disabling hydration::
-
+    $query->enableHydration(false);
```

Comment:
There is an [indentation error here](https://travis-ci.org/cakephp/docs/builds/259911528#L473)